<!DOCTYPE html>
<!--[if IE 8]><html class="no-js lt-ie9" lang="en" > <![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js" lang="en" > <!--<![endif]-->
<head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  
  
  <link rel="shortcut icon" href="../img/favicon.ico">
  <title>lumpy analysis - Mouse Genetics</title>
  <link href='https://fonts.googleapis.com/css?family=Lato:400,700|Roboto+Slab:400,700|Inconsolata:400,700' rel='stylesheet' type='text/css'>

  <link rel="stylesheet" href="../css/theme.css" type="text/css" />
  <link rel="stylesheet" href="../css/theme_extra.css" type="text/css" />
  <link rel="stylesheet" href="../css/highlight.css">
  
  <script>
    // Current page data
    var mkdocs_page_name = "lumpy analysis";
    var mkdocs_page_input_path = "lumpy.md";
    var mkdocs_page_url = "/lumpy/";
  </script>
  
  <script src="../js/jquery-2.1.1.min.js"></script>
  <script src="../js/modernizr-2.8.3.min.js"></script>
  <script type="text/javascript" src="../js/highlight.pack.js"></script> 
  
</head>

<body class="wy-body-for-nav" role="document">

  <div class="wy-grid-for-nav">

    
    <nav data-toggle="wy-nav-shift" class="wy-nav-side stickynav">
      <div class="wy-side-nav-search">
        <a href=".." class="icon icon-home"> Mouse Genetics</a>
        <div role="search">
  <form id ="rtd-search-form" class="wy-form" action="../search.html" method="get">
    <input type="text" name="q" placeholder="Search docs" />
  </form>
</div>
      </div>

      <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="main navigation">
	<ul class="current">
	  
          
            <li class="toctree-l1">
		
    <a class="" href="..">Home</a>
	    </li>
          
            <li class="toctree-l1">
		
    <a class="" href="../seq/">Sequencing Protocol</a>
	    </li>
          
            <li class="toctree-l1">
		
    <a class="" href="../lumpy_approach/">Lumpy approach</a>
	    </li>
          
            <li class="toctree-l1">
		
    <a class="" href="../connection/">Trainee connection</a>
	    </li>
          
            <li class="toctree-l1">
		
    <a class="" href="../import/">Import datasets</a>
	    </li>
          
            <li class="toctree-l1">
		
    <a class="" href="../BWA/">BWA mem tool</a>
	    </li>
          
            <li class="toctree-l1 current">
		
    <a class="current" href="./">lumpy analysis</a>
    <ul class="subnav">
            
    <li class="toctree-l2"><a href="#lumpy-tool-lumpy-step">Lumpy tool - lumpy step</a></li>
    

    </ul>
	    </li>
          
            <li class="toctree-l1">
		
    <a class="" href="../filter_vcf/">Filter VCF file for visualization</a>
	    </li>
          
            <li class="toctree-l1">
		
    <a class="" href="../coverage/">Compute genome coverage</a>
	    </li>
          
            <li class="toctree-l1">
		
    <a class="" href="../workflow/">Generate workflow from history</a>
	    </li>
          
            <li class="toctree-l1">
		
    <a class="" href="../run_workflow/">Run the workflow</a>
	    </li>
          
            <li class="toctree-l1">
		
    <a class="" href="../visualisation/">Visualisation in UCSC Genome Browser</a>
	    </li>
          
        </ul>
      </div>
      &nbsp;
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">

      
      <nav class="wy-nav-top" role="navigation" aria-label="top navigation">
        <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
        <a href="..">Mouse Genetics</a>
      </nav>

      
      <div class="wy-nav-content">
        <div class="rst-content">
          <div role="navigation" aria-label="breadcrumbs navigation">
  <ul class="wy-breadcrumbs">
    <li><a href="..">Docs</a> &raquo;</li>
    
      
    
    <li>lumpy analysis</li>
    <li class="wy-breadcrumbs-aside">
      
    </li>
  </ul>
  <hr/>
</div>
          <div role="main">
            <div class="section">
              
                <h2 id="lumpy-tool-lumpy-step">Lumpy tool - lumpy step</h2>
<p>Now you are going to analyse the BAM alignment file generated by BWA, using lumpy-sv.</p>
<p>The lumpy-sv tool analyse the alignments and search for</p>
<ul>
<li>
<p>split alignment</p>
<p>a read whose 5' and 3' parts map to non-contiguous regions</p>
</li>
<li>
<p>discordant alignments of read pairs</p>
<p>Either one mate maps to one chromosome and the other mate maps to another chromosome, or
the distance between the two mates is beyond what is statistically expected (the library fragment
size in average).  For an insertion, the distance increases, for a deletion, the distance decreases</p>
</li>
</ul>
<p>from, this parsing, lumpy then constructs models of break points that can explain its findings, and report these models
in a vcf file (vcf stands for variant calling format), with statistical significance, number of evidences founds, etc...</p>
<pre><code>1. launch the lumpy-sv tool

2. review the parameters:
    - input(s) : One sample (because we are not comparing the alignments with a reference alignment)
    - One BAM alignment file produced by BWA-mem: be sure to toggle the dataset collection mode
    (arrow in the screen shot below).
    - sequencing method: paired-end sequencing
    - variant calling format: vcf

3. run the tool !
</code></pre>
<p><img alt="" src="../images/lumpy.png" /></p>
<pre><code>4. Look at the vcf returned by lumpy-sv.
</code></pre>
<p>It should be easy to read: lumpy output variation as a suite of single lines (for deletions, insertion, or SNPs), or
as a suite of line pairs, for translocation (one line for the translocation, another line for the reciprocal translocation).
Note that sometimes, there is evidence for one translocation, but not for the reciprocal event. Look at the ID column: a
pair of translocation event model will have for instance IDs and 2_1 and 2_2, respectively.</p>
<p>However, the lumpy vcf format is not suitable for visualisation in a genome browser such as UCSC genome browser.
For these browser, lines have to be sorted in the order of chromosomes (first) and in the order of coordinates (secondly).</p>
<p>The next steps of the analysis are for reorganise the lumpy vcf output to follow these rules. Also, we are going to focus
on translocations, filtering out other variation we are not interested in.</p>
              
            </div>
          </div>
          <footer>
  
    <div class="rst-footer-buttons" role="navigation" aria-label="footer navigation">
      
        <a href="../filter_vcf/" class="btn btn-neutral float-right" title="Filter VCF file for visualization">Next <span class="icon icon-circle-arrow-right"></span></a>
      
      
        <a href="../BWA/" class="btn btn-neutral" title="BWA mem tool"><span class="icon icon-circle-arrow-left"></span> Previous</a>
      
    </div>
  

  <hr/>

  <div role="contentinfo">
    <!-- Copyright etc -->
    
  </div>

  Built with <a href="http://www.mkdocs.org">MkDocs</a> using a <a href="https://github.com/snide/sphinx_rtd_theme">theme</a> provided by <a href="https://readthedocs.org">Read the Docs</a>.
</footer>
      
        </div>
      </div>

    </section>

  </div>

  <div class="rst-versions" role="note" style="cursor: pointer">
    <span class="rst-current-version" data-toggle="rst-current-version">
      
      
        <span><a href="../BWA/" style="color: #fcfcfc;">&laquo; Previous</a></span>
      
      
        <span style="margin-left: 15px"><a href="../filter_vcf/" style="color: #fcfcfc">Next &raquo;</a></span>
      
    </span>
</div>
    <script>var base_url = '..';</script>
    <script src="../js/theme.js"></script>
      <script src="../search/require.js"></script>
      <script src="../search/search.js"></script>

</body>
</html>
